server/testkit/Cargo.toml

```diff
@@ -23,27 +23,35 @@ default = []
 webdriver = ["fantoccini"]
 
 [dependencies]
+assert_cmd = "2.0.12"
+escargot = "0.5.8"
+# used for webdriver testing
+fantoccini = { version = "0.19.3", optional = true }
```

Comment:
If it's for testing, could this be in dev-dependencies.

Reply:
Will try that, I think it was being a jerk about things at one point.